src/lib/types.d.ts

```diff
@@ -1,4 +1,4 @@
-import { ChatCompletionMessageParam, ChatCompletionAssistantMessageParam, ChatCompletionUserMessageParam } from "@mlc-ai/web-llm";
+import type { ChatCompletionMessageParam, ChatCompletionAssistantMessageParam, ChatCompletionUserMessageParam } from "@mlc-ai/web-llm";
```

Comment:
using `import type` so it's only for type checking and not part of the actual imports